src/redux/migrations.test.ts

```diff
@@ -492,4 +493,14 @@ describe('Redux persist migrations', () => {
 
     expect(migratedSchema).toMatchObject(expectedSchema)
   })
+
+  it('works for v50 to v51', () => {
```

Comment:
this is fine of course, but to save effort in the future i think we only need to write tests for complex migrations (e.g. if new properties need to have specific values, or we are deriving new properties based on previous state, etc) - the simple migrations for adding a new property is implicitly covered by the test [here](https://github.com/valora-inc/wallet/blob/f6ac12e612938e3936df5d79ab3ac0bca8a0cca6/src/redux/createMigrate.test.ts#L19)